guides/v2.3/ui_comp_guide/components/ui-image-preview.md

```diff
@@ -0,0 +1,45 @@
+---
+group: ui-components-guide
+title: ImagePreview Component
+contributor_name: Shankar Konar
+contributor_link: https://github.com/konarshankar07
+---
+
+The ImagePreview component implements a detailed view image column. User can preview next or previous image by clicking next or previous arrow
```

Comment:
Thanks for the contribution. Can you please clarify this sentence? I don't have a good idea of what this component actually does.

Reply:
Can you include a screen capture of this component?

Reply:
Perhaps we could expand this introductory sentence a bit more. Here's my stab at it:

```suggestion
The ImagePreview component implements an inline detailed preview of an image, which is typically used in conjunction with a grid of image search results. Users can expand a more detailed preview of an image in-line with other image search results and also have the ability to click on "next" or "previous" arrows to navigate to the next or previous image's detailed preview.
```